R/generate_retro_triangles.R

```diff
@@ -0,0 +1,108 @@
+#' Generate retrospective reporting triangles
+#'
+#' This function ingests a reporting triangle and the number of retrospective
+#'   reporting triangles we want to create, and iteratively generates the
+#'   reporting triangle that would have been available as of the maximum
+#'   reference time, and iteratively working backwards for
+#'   `n_history_uncertainty` snapshots
```

Comment:
is this an out of date argument?

Reply:
Yes let me fix, this should be `n_triangles`, or `n` as discussed above